docs/user-guide/amor/amor-reduction.ipynb

```diff
@@ -18,6 +18,16 @@
     "## Setup"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import warnings\n",
+    "warnings.filterwarnings('ignore', message=r'Failed to convert .* into a transformation')"
```

Comment:
Sorry to be a pain, but I think this should not be visible to users in the notebook.
It just adds noise and confusion for users,

Instead, we should filter the warnings in the underlying library code that produces them.

Reply:
That makes sense. I'll move the filter to the `amor.load` module